### PR TITLE
IGNITE-11709 .NET thin client: introduce Cluster API

### PR DESCRIPTION
**Pull request by ashapkin**

### CODE REVIEW

File: modules/core/src/main/java/org/apache/ignite/internal/processors/platform/client/cluster/ClientClusterIsActiveRequest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+     * Constructor.
+     *
+     * @param reader Reader/
+     */
+    public ClientClusterIsActiveRequest(BinaryRawReader reader) {
+        super(reader);
+    }
+
+    /** {@inheritDoc} */
+    @Override
```
Comment (ptupitsyn) [inline modules/core/src/main/java/org/apache/ignite/internal/processors/platform/client/cluster/ClientClusterIsActiveRequest.java]: Here and in other places, `@Override` should be on the same line

File: modules/core/src/main/java/org/apache/ignite/internal/processors/platform/client/cluster/ClientClusterChangeStateRequest.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ * Cluster status request.
+ */
+public class ClientClusterChangeStateRequest extends ClientRequest {
+    /** Next state. */
+    private final boolean isActive;
+
+    /**
+     * Constructor.
+     *
+     * @param reader Reader/
```
Comment (ptupitsyn) [inline modules/core/src/main/java/org/apache/ignite/internal/processors/platform/client/cluster/ClientClusterChangeStateRequest.java]: Here and below - comment should not end with `/`

File: modules/core/src/main/java/org/apache/ignite/internal/processors/platform/client/cluster/ClientClusterWalChangeStateResponse.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    /**
+     * Operation result.
+     */
+    private final boolean res;
+
+    /**
+     * Ctor.
+     *
+     * @param reqId Request id.
+     * @param res    Operation result.
```
Comment (ptupitsyn) [inline modules/core/src/main/java/org/apache/ignite/internal/processors/platform/client/cluster/ClientClusterWalChangeStateResponse.java]: Please remove extra spaces

File: modules/core/src/main/java/org/apache/ignite/internal/processors/platform/client/cluster/ClientClusterIsActiveRequest.java (hunk 1 above)
Comment (ashapkin) [reply modules/core/src/main/java/org/apache/ignite/internal/processors/platform/client/cluster/ClientClusterIsActiveRequest.java]: Thanks, fixed

**Pull request merged**